scripts/vaccination_alerts/generate-users-to-email.ts

```diff
@@ -84,7 +108,14 @@ async function main() {
 }
 
 if (require.main === module) {
-  main()
+  const argv = yargs.options({
```

Comment:
I think we need to add yargs to the dependencies of the project

Reply:
yeah I think it's pulled in as a dependency but definitely should be explicitly defined - will do